### PR TITLE
Feature/mx 1556 extract grippe web data

### PR DESCRIPTION
**Pull request by erichesse**
# PR Context

# Added
- grippeweb extract

# Changes
- move MSSQL Server authentication to general settings

# Deprecated

# Removed

# Fixed

# Security

### CODE REVIEW

**Review by mr-kamran-ali — COMMENTED** [review]
Nicely done, nothing major to fix just few clarifications. 

Tests are all clear, but I get `..Documents\\workspace\\mex-assets\\mappings\\__final__\\test_mapping\\access-platform.yaml` when I run grippeweb. I have already merged your `mex-assets` grippeweb branch and decrypted fresh envs. Am I missing some mapping files somehow in my `mex-assets`?

**Comment by erichesse** [comment]
> Nicely done, nothing major to fix just few clarifications.
> 
> Tests are all clear, but I get `..Documents\\workspace\\mex-assets\\mappings\\__final__\\test_mapping\\access-platform.yaml` when I run grippeweb. I have already merged your `mex-assets` grippeweb branch and decrypted fresh envs. Am I missing some mapping files somehow in my `mex-assets`?

I couldn't reproduce this error. But actually, the mex-assets settings still needed to be changed to __final_ folder. Can you try that out? And if your error persists, maybe we can have a look at it together?
https://bitbucket.rki.local/projects/MF4/repos/mex-assets/pull-requests/152/overview

**Comment by mr-kamran-ali** [comment]
> I couldn't reproduce this error. But actually, the mex-assets settings still needed to be changed to __final_ folder. Can you try that out? And if your error persists, maybe we can have a look at it together? https://bitbucket.rki.local/projects/MF4/repos/mex-assets/pull-requests/152/overview

so if I run `pdm run grippeweb` and put a breakpoint at this line: `https://github.com/robert-koch-institut/mex-extractors/blob/5924b502f347138598c8d8aaffab591d34992c9f/mex/mapping/extract.py#L21`

I get `WindowsPath('C:/Users/alik/Documents/workspace/mex-assets/mappings/__draft__/grippeweb/access-platform.yaml')`

`__draft__` is there instead of __final__, where is it coming from? and why isnt it going to __final__ by default?

**Comment by erichesse** [comment]
> > I couldn't reproduce this error. But actually, the mex-assets settings still needed to be changed to __final_ folder. Can you try that out? And if your error persists, maybe we can have a look at it together? https://bitbucket.rki.local/projects/MF4/repos/mex-assets/pull-requests/152/overview
> 
> so if I run `pdm run grippeweb` and put a breakpoint at this line: `https://github.com/robert-koch-institut/mex-extractors/blob/5924b502f347138598c8d8aaffab591d34992c9f/mex/mapping/extract.py#L21`
> 
> I get `WindowsPath('C:/Users/alik/Documents/workspace/mex-assets/mappings/__draft__/grippeweb/access-platform.yaml')`
> 
> `__draft__` is there instead of `__final__`, where is it coming from? and why isnt it going to `__final__` by default?

it comes from the `.env`, we should merge https://bitbucket.rki.local/projects/MF4/repos/mex-assets/pull-requests/152/overview
then pull and decrypt again

File: mex/grippeweb/connector.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+    def connect(self, _: str) -> None:  # pragma: no cover
+        """Create a new ODBC connection to a database."""
+        return
+
+
+try:
+    import pyodbc  # type: ignore[import-not-found]
+except ImportError:
+    pyodbc = NoOpPyodbc
```
[the remark this answers is not in the document]
Comment (erichesse) [reply mex/grippeweb/connector.py]: https://github.com/robert-koch-institut/mex-extractors/pull/36/commits/33efb0dda3298743584e6aed81d484ca21bf5665

**Review by mr-kamran-ali — APPROVED** [review]
Very nice work, merge away!

**Pull request merged**